storage/innobase/xtrabackup/doc/source/index.rst

```diff
@@ -93,17 +110,76 @@ Advanced Features
 
    advanced/throttling_backups
    advanced/encrypted_innodb_tablespace_backups
-   advanced/locks
+   xtrabackup_bin/backup.encrypting
+   xtrabackup_bin/lru_dump
+   xtrabackup_bin/point-in-time-recovery
+   xtrabackup_bin/working_with_binary_logs
+
+
+Auxiliary guides
```

Comment:
This title is in sentence style, while all others are in title case.

Reply:
Updated.